### PR TITLE
moved storage to util in internal, ref binding

### PR DESCRIPTION
**Pull request by 6missedcalls**
Code cleanup and refactor

## Changes

- Addition of comment explanations for the binding layer
- Moved store & load from storage into utils 
- Updated Readme
- Added QueryByCreator Routes for Schemas and Buckets.
- Taskfile Additions
  - build:ui:dev - builds to the /out directory
  - build:ui:debug - build to the /server/build directory

### New Features *(required)*
N/A

### Deprecations *(required)*
N/A

## Checklist

- [ ] Unit tests
- [ ] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

**Comment by tgfassina** [comment]
Hey @6missedcalls , if we merge it like this, then it's gonna break the UI because the proxy endpoints are being deleted but they are still in use by the UI. Can we keep them until I do the UI part? Then it won't be used anymore and I can commit the cleanup

File: server/routes/query-bucket-by-creator.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	})
+	if err != nil {
+		// TODO: Add proper error message
+		c.JSON(http.StatusInternalServerError, FailedResponse{
+			Error: err.Error(),
+		})
+		return
+	}
+
+	c.JSON(http.StatusOK, rtmv1.QueryWhereIsByCreatorResponse{
```
Comment (tgfassina) [inline server/routes/query-bucket-by-creator.go:49]: Just a reminder to see if we can return the whole response of `QueryWhereIsByCreator` like we talked before

**Review by tgfassina — APPROVED** [review]
thanks Ian!

File: internal/binding/binding.go [hunk 2]
```diff
@@ -13,20 +13,19 @@ import (
...
 
 var (
-	ErrMotorNotInitialized = fmt.Errorf("cannot find instance of motor")
-	ErrNotAuthenticated    = fmt.Errorf("must be logged in to preform that action")
-	ErrIsAuthenticated     = fmt.Errorf("you are already authenticated")
-	ErrMotorFailedInit     = fmt.Errorf("motor failed to initialize")
+	ErrMotorNotInitialized = fmt.Errorf("cannot find instance of motor")            // Error for when the motor is not initialized
+	ErrNotAuthenticated    = fmt.Errorf("must be logged in to preform that action") // Error for when the user is not logged in
+	ErrIsAuthenticated     = fmt.Errorf("you are already authenticated")            // Error for when the user is already logged in
+	ErrMotorFailedInit     = fmt.Errorf("motor failed to initialize")               // Error for when the motor fails to initialize
```
Comment (joshLong145) [inline internal/binding/binding.go:23]: we can remove the comments as the error strings are descriptive of the purpose

**Review by joshLong145 — CHANGES_REQUESTED** [review]

**Review by joshLong145 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -115,7 +132,7 @@ To build for dev, run `task server:start:debug` instead of `task server:start:re
 ### Running the UI: Mock Server Mode
 This mode is ideal for UI development. Changes to the code will reflect on the browser automatically and there is no need to build. Any data generated will be local only. Behavior may not perfectly match the Sonr Speedway Backend.
 
-**Tun the Frontend:**
```
Comment (ntindle) [inline README.md:118]: oof

**Review by ntindle — APPROVED** [review]

**Pull request merged**